tests/drivers/counter/counter_basic_api/boards/imx8mm_evk_mimx8mm6_a53.conf

```diff
@@ -0,0 +1 @@
+CONFIG_COUNTER_MCUX_GPT=y
```

Comment:
Is this needed? I'd assume this Kconfig should be enabled automatically since the GPT node has `status=okay`. Same comment would apply to other Kconfig overlays

Reply:
Removed, I just get this feature that Kconfig can be enabled by dts, thanks.